addons/website_blog/controllers/main.py

```diff
@@ -27,7 +27,9 @@ def tags_list(self, tag_ids, current_tag):
             tag_ids.remove(current_tag)
         else:
             tag_ids.append(current_tag)
-        tag_ids = request.env['blog.tag'].browse(tag_ids).exists()
+        # TODO: can we assume tag_ids exists since it comes from `active_tag_ids`
+        # in controller where they got already checked with exists()?
+        tag_ids = request.env['blog.tag'].browse(tag_ids)
```

Comment:
since slug will make extra request to have name get of each blog... 
We should probably just generate link tag_ids = 1,2,3 and let the controller make redirect iff we click on it.
what you think about ?

Reply:
It actually does not make a request to get display_name in slug as it is already prefetched by the controller so I think we are good here